src/lib/stable/canister_methods/index.ts

```diff
@@ -264,3 +344,46 @@ function isDecoratorOverloadedWithoutParams<
         param2.name !== undefined
     );
 }
+
+/**
+ * @internal
+ *
+ * Uses the candid string of the init and post-upgrade methods to verify that
```

Comment:
Use backticks and say `init` and `postUpgrade`

Reply:
Done